7.4-alpine/Dockerfile

```
@@ -60,6 +60,11 @@ EXPOSE 80
 
 FROM web as rootless
 
+# https://github.com/docker-library/httpd/issues/118
+# para soportar Docker <= 19.x
+RUN apk add libcap
```

Le agregaria el `no-cache` pa que no quede dando vueltas esa info en la imagen final.

```suggestion
RUN apk --no-cache add libcap
```